doc/developer_guide/coding-style.rst

```diff
@@ -155,6 +155,47 @@ retained.). For example::
     except KeyError as err:
         raise InternalError("Useful message here") from err
 
+
+Logging
+-------
+
+PSyclone supports the Python `logging` module, to help enable improved
+general messages throughout the code. By default the logging infrastructure
+is switched off, but can be enabled either through the PSyclone script
+options or programatically by::
```

Comment:
s/by/with/ ?

Also you could use ".. code-block:: python" instead of "::" for proper syntax colouring (here and below)

Reply:
Yeah, changed.